package.json

```diff
@@ -168,6 +170,7 @@
 		"rimraf": "3.0.2",
 		"rtlcss": "2.4.0",
 		"sass-loader": "6.0.7",
+		"selenium-webdriver": "3.6.0",
```

Comment:
Are these large dependencies?

See also: #15667

Reply:
According to [bundlephobia](https://bundlephobia.com/result?p=selenium-webdriver@3.6.0), it's 362.3kb. I opened up property window and it's only 3.31MB. I don't think it's big, right?

![image](https://user-images.githubusercontent.com/8130013/79087096-4166ea80-7d79-11ea-8eb6-7576066c04a8.png)

Reply:
That's fine. I'm more concerned if it was on the scale of Puppeteer (100MB+). I guess the difference is that Puppeteer is downloading the actual browser executables, whereas here we aren't.